composer/algorithms/mixup/README.md

```diff
@@ -169,7 +174,13 @@ Doing so avoids putting additional work on the CPU (since augmentation occurs on
 
 >❗ When `interpolate_loss=True` MixUp interpolates the loss rather than the targets.
 >
-> This is fine for loss functions that are linear in the targets, such as cross entropy, but may produce unexpected results for other loss functions.
+> This is fine for loss functions that are linear in the targets, such as cross entropy,
+> but may produce unexpected results for other loss functions.
+
+> 🚧 MixUp May Interact with Other Methods that Modify Targets
+>
+> This method interacts with other methods (such as Label Smoothing) that alter the targets.
+> While such methods may still compose well with label smoothing in terms of improved accuracy, it is important to ensure that the implementations of these methods compose.
```

Comment:
Does composer not take care of this? wording doesn't make clear what I have to do differently / worry about as a user

Reply:
Good question. I saw it in the label smoothing readme, so I added it to this. @coryMosaicML, do you have any insights here?

Reply:
Composer takes care of this if everything is run with one hot rather than index labels. However, by popular demand MixUp and CutMix have an index label mode that can be used. The bad interaction here is when MixUp and CutMix are both used in index label mode (`interpolate_loss=True`), which will not compose correctly. There used to be a warning that got raised if someone tried to do this, but it looks like it got removed at some point?!